src/App.js

```diff
@@ -22,9 +21,8 @@ class App extends React.Component {
     return (
       <div>
         {/* TODO: Put signout feature in layout component */}
-        <AmplifySignOut />
+        {/* <AmplifySignOut /> */}
```

Comment:
Will have to move the AmplifySignOut component to display when we select the user profile icon in a dialog potentially

Reply:
This is fine.  No need to have the sign out button here at this time